pandas/tests/arrays/test_period.py

```diff
@@ -323,3 +324,91 @@ def test_min_max_empty(self, skipna):
 
         result = arr.max(skipna=skipna)
         assert result is pd.NaT
```

Comment:
i would put the pyarrow tests in a separate file

Reply:
Do you mean for all different dtypes combined? (because the period tests on which you are commenting here is less than 100 lines of code, so not sure that it is worth a separate file)